hera_pspec/utils.py

```diff
@@ -895,7 +910,11 @@ def get_reds(uvd, bl_error_tol=1.0, pick_data_ants=False, bl_len_range=(0, 1e4),
         List of bad antenna numbers to exclude
 
     add_autos : bool
-        If True, add into autocorrelation to redundant groups
+        If True, add into autocorrelation group to the redundant group list.
+
+    low_hi : bool
```

Comment:
Bit of a strange option...

Reply:
Yeah.. it turns out to be a convenience option based on how `redcal.get_reds` works...